docs/user/aws/install_upi.md

```diff
@@ -0,0 +1,432 @@
+openshift-operator-lifecycle-manager                    olm-operators-zxl6x                                                 1/1       Running     0          35m
+openshift-operator-lifecycle-manager                    packageserver-65dbff8945-2vm95                                      1/1       Running     0          35m
+openshift-operator-lifecycle-manager                    packageserver-65dbff8945-j8msv                                      1/1       Running     0          35m
+openshift-sdn                                           ovs-nsjrm                                                           1/1       Running     0          36m
+openshift-sdn                                           ovs-qpgs8                                                           1/1       Running     0          36m
+openshift-sdn                                           ovs-tn49d                                                           1/1       Running     0          36m
+openshift-sdn                                           sdn-controller-4m2xs                                                1/1       Running     1          36m
+openshift-sdn                                           sdn-controller-rd6ql                                                1/1       Running     0          36m
+openshift-sdn                                           sdn-controller-w2rht                                                1/1       Running     0          36m
+openshift-sdn                                           sdn-crzgm                                                           1/1       Running     0          36m
+openshift-sdn                                           sdn-p5mkf                                                           1/1       Running     1          36m
+openshift-sdn                                           sdn-wltn4                                                           1/1       Running     0          36m
+openshift-service-ca-operator                           openshift-service-ca-operator-66ff6dc6cd-9r257                      1/1       Running     0          37m
+openshift-service-ca                                    apiservice-cabundle-injector-695b6bcbc-cl5hm                        1/1       Running     0          35m
+openshift-service-ca                                    configmap-cabundle-injector-8498544d7-25qn6                         1/1       Running     0          35m
+openshift-service-ca                                    service-serving-cert-signer-6445fc9c6-wqdqn                         1/1       Running     0          35m
+openshift-service-catalog-apiserver-operator            openshift-service-catalog-apiserver-operator-549f44668b-b5q2w       1/1       Running     0          32m
+openshift-service-catalog-controller-manager-operator   openshift-service-catalog-controller-manager-operator-b78cr2lnm     1/1       Running     0          31m```
+
```

Comment:
You need a terminal backtick fence here:

    ```

Reply:
Fixed.